examples/hf_serialization.py

```diff
@@ -304,6 +386,14 @@ def main():
         action="store_true",
         help="Validate serialization by running a test inference",
     )
+    parser.add_argument(
+        "--force",
+        action="store_true",
+        help=(
+            "Force upload serialized tensors to output_prefix even if they"
+            " already exist"
+        ),
+    )
```

Comment:
We should print a diagnostic message if serialization is skipped due to this flag being disabled so people can tell what is going on.

Reply:
Will do so and resolve once fixed.

Reply:
Resolved in #76